recipes/clcache/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "clcache" %}
+{% set version = "3.3.0" %}
+{% set sha256 = "57659520283809b96035bc1a39678588df1143435e3b3467f1fe82d564d93ad4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/frerich/clcache/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  entry_points:
+    - clcache = clcache:main
+  skip: True  # [not win or py<34]
+
+requirements:
+  run:
+    - python
```

Comment:
I don't think that VC features are necessary here since this is a python-only package. There is no compilation.

About it not building: I don't know why it is not building. The `skip` section reads like this: `skip: True  # [not win or py<34]`. Why wouldn't it build?
Yes, I saw that @jakirkham